tests/workload/apis.go

```diff
@@ -30,3 +31,15 @@ func GetNetwork(ctx context.Context, require *require.Assertions, uris []string,
 		require.Equal(expectedChainID, chainID)
 	}
 }
+
+func ReadState(ctx context.Context, require *require.Assertions, uris []string) {
+	for _, uri := range uris {
+		client := state.NewJSONRPCStateClient(uri)
+		values, readerrs, err := client.ReadState(ctx, [][]byte{
+			[]byte(`my-unknown-key`),
+		})
+		require.NoError(err)
+		require.Len(values, 1)
+		require.Len(readerrs, 1)
```

Comment:
Hmm it's a bit of a strange problem here that we don't know any other key-value pairs that could be included in the state, so this is all we can test.

Could we move this directly into `e2e.go` ? The functions here are intended for re-use across environments and I don't think that we'll re-use this test in any other environment.